Formula/cpl.rb

```diff
@@ -0,0 +1,36 @@
+class Cpl < Formula
+  desc "ISO-C libraries for developing astronomical data-reduction tasks"
+  homepage "https://www.eso.org/sci/software/cpl/index.html"
+  url "ftp://ftp.eso.org/pub/dfs/pipelines/libraries/cpl/cpl-7.1.tar.gz"
+  sha256 "ba7f762fac3550e9d9246b7fbacba54bfba4c47b833720834bd4cd21ed0e8b00"
+
+  depends_on "cfitsio"
+  depends_on "fftw"
+  depends_on "wcslib"
+
+  conflicts_with "gdal", :because => "both install cpl_error.h"
+
+  def install
+    wcslib = Formula["wcslib"]
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--with-wcslib=#{wcslib.prefix}"
```

Comment:
This might be cleaner as the variable is only used once
```suggestion
                          "--with-wcslib=#{Formula["wcslib"].prefix}"
```